WordPressUI/WordPressUI/Extensions/Events without selectors/UIBarButtonItem+BlockEvents.swift

```diff
@@ -0,0 +1,36 @@
+// Adding UIBarButtonItem
```

Comment:
We probably want to include the reference to the GitHub repo as well as the license here as well.

Reply:
I wrote that code, I assume it's under our license.

Reply:
haha under Nate's license, for some reason I thought it all came from the same repo, sorry! 🤦‍♀️